src/index.ts

```diff
@@ -52,7 +52,7 @@ jQuery.when(window.mw.loader.using('mediawiki.util'), $.ready).then((): void =>
     'ko': i18nSource['korean'],
     'yue': i18nSource['cantonese'],
     'zh-yue': i18nSource['cantonese'],
-    'zh': i18nSource['traditional_chinese'],
+    'zh': i18nSource['simplified_chinese'],
```

Comment:
I don't understand, can you give the source code of the corresponding practice? But keeping Traditional Chinese is just my personal preference.

Reply:
(Inheritance Diagram)[https://doc.wikimedia.org/mediawiki-core/master/php/classLanguageZh.html] from WMF docs.

Reply:
I mean, you may again request the variant the user is using but from my experience most of the gadgets just read from the interface language.

Reply:
Yes, this **shall** follow the user interface language.